modules/custom/social_advanced_image/social_advanced_image.install

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the social_advanced_image module.
+ */
+
+use Drupal\Core\Database\Database;
+
+/**
+ * Implements hook_install().
+ *
+ * Perform actions on install related to the image copyright fields.
+ */
+function social_advanced_image_install() {
+  $spec = [
+    'type' => 'varchar',
+    'description' => "Image copyright text, for the image's 'copyright' attribute.",
+    'length' => 1024,
+  ];
+
+  $schema = Database::getConnection()->schema();
+  $entity_definitions = \Drupal::entityDefinitionUpdateManager();
+  $change_list = $entity_definitions->getChangeList();
+  if (!empty($change_list['node']['field_storage_definitions'])) {
+    // Check all changed definitions and update the tables.
+    foreach ($change_list['node']['field_storage_definitions'] as $field_name => $value) {
+      // Update field table.
+      if (!$schema->fieldExists('node__' . $field_name, $field_name . '_copyright')) {
+        $schema->addField('node__' . $field_name, $field_name . '_copyright', $spec);
+      }
+      // Update revisions table.
+      if (!$schema->fieldExists('node_revision__' . $field_name, $field_name . '_copyright')) {
+        $schema->addField('node_revision__' . $field_name, $field_name . '_copyright', $spec);
+      }
+    }
+  }
+}
+
+
+/**
+ * Implements hook_uninstall().
+ *
+ * Perform actions on uninstall related to the image copyright fields.
+ */
+function social_advanced_image_uninstall() {
+
```

Comment:
In here I think we need to remove the fields from the node table schemas still.

Reply:
Yeah I know, it's on the todo of this PR.